azure-tools-common/src/main/java/com/microsoft/azure/common/function/configurations/Retry.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.common.function.configurations;
+
+import com.microsoft.azure.functions.annotation.ExponentialBackoffRetry;
+import com.microsoft.azure.functions.annotation.FixedDelayRetry;
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder(toBuilder = true)
+@Data
+public class Retry {
```

Comment:
are its setXXX() methods used in anywhere? if not, I prefer to user @Getter and @ToString to replace with @Data.